theme/templates/includes/navbar.html

```diff
@@ -337,12 +337,12 @@ <h4>Notifications <span v-if="tasks.length">(${tasks.length})</span></h4>
                                 <li>
                                     <a data-toggle="tooltip" data-placement="left"
                                        title="A shortcut (or reference) to an external web application that functions with HydroShare data resources."
-                                       data-modal-title="Create a Web App Connector"
-                                       data-modal-input-title="Enter the name for the new Web App Connector."
+                                       data-modal-title="Create a App Connector"
+                                       data-modal-input-title="Enter the name for the new app connector."
```

Comment:
caps?

Reply:
I changed it to lower case to be consistent with resource and collection modals.

Reply:
makes sense